### ISSUE TITLE
Theme: color contrast issues

### ISSUE THREAD

**Issue by Heydon** [iTwin/stratakit#1241]
# 1. Failed contrast pairings

<table>
    <tr>
        <th>color 1</th>
        <th>color 2</th>
        <th>ratio</th>
        <th>screenshot</th>
        <th>notes</th>
    </tr>
    <tr>
        <td><code>--stratakit-color-bg-page-base (oklch(99.06% .002 247.84));</code></td>
        <td><code>--variant-outlinedBorder (oklch(from var(--stratakit-mui-palette-secondary-main) l c h / 0.5))</code></td>
        <td>1.95</td>
        <td>

<img width="279" height="122" alt="Image" src="https://github.com/user-attachments/assets/f298f90b-734f-4c50-a529-0ad371b12182" />

</td>
        <td>This is a non-text contrast issue between the page background and the outlined button border</td>
    </tr>
    <tr>
        <td><code>--stratakit-mui-shadows-2 (oklch(0 none none / 0.04) 0px 1px 0px 0px, oklch(0 none none / 0.08) 0px 0px 0px 1px, oklch(1 none none / 0.08) 0px 0px 0px 1px inset, oklch(0 none none / 0.04) 0px 1px 1px -0.5px, oklch(0 none none / 0.04) 0px 3px 3px -1.5px, oklch(0 none none / 0.04) 0px 6px 6px -3px);</code></td>
        <td><code>--variant-outlinedBorder (oklch(from var(--stratakit-mui-palette-secondary-main) l c h / 0.5))</code></td>
        <td>1.08</td>
<td>

<img width="324" height="131" alt="Image" src="https://github.com/user-attachments/assets/3d5cc84c-f865-4f22-9547-d600722f54fb" />

</td>
        <td>This is a non-text contrast issue between the page background and the contained button <code>box-shadow</code>. This is not a border style but a <code>box-shadow</code> that appears as a border. This also creates a misalignment with outlined buttons (which use border)</td>
    </tr>
    <tr>
        <td><code>--stratakit-color-bg-page-base (oklch(99.06% .002 247.84));</code></td>
        <td><code>--stratakit-color-border-page-base (oklch(0% none none / .08)</code></td>
        <td>1.16</td>
<td>

<img width="229" height="131" alt="Image" src="https://github.com/user-attachments/assets/b6523fae-fd9e-4f66-8b1e-cc26bf9f548a" />

</td>
        <td>For cards, <code>--stratakit-color-border-page-base</code> is aliased to <code>--stratakit-mui-palette-divider</code></td>
    </tr>
    <tr>
        <td><code>--stratakit-color-bg-page-base (oklch(99.06% .002 247.84));</code></td>
        <td><code>--stratakit-mui-palette-Chip-defaultBorder (oklch(87.77% .005 247.88)</code></td>
        <td>1.16</td>
<td>

<img width="149" height="61" alt="Image" src="https://github.com/user-attachments/assets/9255e27f-510d-4698-862a-474ddcd18339" />

</td>
        <td>The Chip border color is aliased from <code>--stratakit-color-border-neutral-base:</code></td>
    </tr>
</table>

**Comment by mayank99** [comment]
These contrast issues have been addressed in [@stratakit/mui@**0.5.4**](https://github.com/iTwin/stratakit/releases/tag/%40stratakit%2Fmui%400.5.4), with further improvements planned for future releases.

More details in https://github.com/iTwin/stratakit/pull/1497 and https://github.com/iTwin/stratakit/discussions/701.
